diff --git a/default-plugins/status-bar/src/second_line.rs b/default-plugins/status-bar/src/second_line.rs
index ccab42df..c6cf44f3 100644
--- a/default-plugins/status-bar/src/second_line.rs
+++ b/default-plugins/status-bar/src/second_line.rs
@@ -46,7 +46,7 @@ fn full_length_shortcut(
 }

 fn locked_interface_indication(palette: Palette) -> LinePart {
-    let locked_text = " -- INTERFACE LOCKED -- ";
+    let locked_text = " 🔒 ";
     let locked_text_len = locked_text.chars().count();
     let text_color = palette_match!(match palette.theme_hue {
         ThemeHue::Dark => palette.white,
@@ -467,7 +467,7 @@ pub fn locked_fullscreen_panes_to_hide(palette: &Palette, panes_to_hide: usize)
     });
     let green_color = palette_match!(palette.green);
     let orange_color = palette_match!(palette.orange);
-    let locked_text = " -- INTERFACE LOCKED -- ";
+    let locked_text = " 🔒 ";
     let shortcut_left_separator = Style::new().fg(text_color).bold().paint(" (");
     let shortcut_right_separator = Style::new().fg(text_color).bold().paint("): ");
     let fullscreen = "FULLSCREEN";
@@ -506,7 +506,7 @@ pub fn locked_floating_panes_are_visible(palette: &Palette) -> LinePart {
     };
     let shortcut_left_separator = Style::new().fg(white_color).bold().paint(" (");
     let shortcut_right_separator = Style::new().fg(white_color).bold().paint(")");
-    let locked_text = " -- INTERFACE LOCKED -- ";
+    let locked_text = " 🔒 ";
     let floating_panes = "FLOATING PANES VISIBLE";

     let len = locked_text.chars().count() + floating_panes.chars().count();
